src/main/java/uk/nhs/digital/nhsconnect/nhais/mesh/MeshClient.java

```diff
@@ -0,0 +1,26 @@
+package uk.nhs.digital.nhsconnect.nhais.mesh;
+
+import org.springframework.stereotype.Component;
+
+import java.util.Collections;
+import java.util.List;
+
+@Component
+public class MeshClient {
+
+    // TODO: stub, to be replaced by NIAD-266
+    public void sendEdifactMessage(String messageContent, String recipient) {
+
+    }
+
+    // TODO: stub, to be replaced by NIAD-265
+    public List<String> getInboxMessageIds() {
+        return Collections.emptyList();
```

Comment:
For testing purpose this should return a singleton list with a const message id

Reply:
tests use mocks, doesn't matter